client/jetpack-cloud/sections/partner-portal/company-details-form/index.tsx

```diff
@@ -76,7 +77,9 @@ export default function CompanyDetailsForm( {
 	const [ contactPerson, setContactPerson ] = useState( initialValues.contactPerson ?? '' );
 	const [ companyWebsite, setCompanyWebsite ] = useState( initialValues.companyWebsite ?? '' );
 	const [ companyType, setCompanyType ] = useState( initialValues.companyType ?? '' );
-	const [ managedSites, setManagedSites ] = useState( initialValues.managedSites ?? '' );
+	const [ managedSites, setManagedSites ] = useState(
+		initialValues.managedSites ?? ( showSignupFields ? '1-5' : '' )
```

Comment:
A theoretical micro-optimisation would be to pass a initialisation function here rather than doing the comparison on each render even when not needed, but not a blocker.